### PR TITLE
V5.4.4

### PR DESCRIPTION
**Pull request by Vikrant-UXCAM**
Updated to transitive dependency and updated Android version to 3.5.1. 
Also Upgraded android SDK version in gradle

### CODE REVIEW

File: UXCamReactExample/android/build.gradle [hunk 1]
```diff
@@ -4,8 +4,8 @@ buildscript {
     ext {
         buildToolsVersion = "28.0.3"
         minSdkVersion = 21
-        compileSdkVersion = 28
-        targetSdkVersion = 28
+        compileSdkVersion = 31
+        targetSdkVersion = 31
```
Comment (rubin22) [inline UXCamReactExample/android/build.gradle:8]: Seems these variables are not used anywhere. Please confirm if these are still needed before merging.

Comment (rubin22) [reply UXCamReactExample/android/build.gradle:8]: If these are being used in build.gradle then the version are different with these and that in build.gradle

**Review by rubin22 — APPROVED** [review]
Something to consider before merging

File: UXCamReactExample/android/build.gradle (hunk 1 above)
Comment (Anup-UXCam) [reply UXCamReactExample/android/build.gradle:8]: This change was in sample app UXCamReactExample to support React native versions 0.7+ which we used for testing.

**Pull request merged**